apps/guardian-ui/src/types.tsx

```diff
@@ -47,3 +47,53 @@ export interface Versions {
     }
   >;
 }
+
+export enum ModuleKind {
+  Ln = 'ln',
+  Mint = 'mint',
+  Wallet = 'wallet',
+}
+
+interface FedimintModule {
+  config: string;
```

Comment:
This seems like it will be an interesting one to handle on the front end. How does the string representation look? Is it human readable?

Reply:
Oh this is a hash string. We'd need to decode it to get front-end usable JSON.

Example of config for LN module: `b0fee25a8e3758c11244dfa892243283e620581f94a6a14cf516210685ea978a1872832d3592812c6353295bc2885b6a00000000000000000000000000000000dab5bffa`